os/include/tinyara/input/touchscreen.h

```diff
@@ -121,7 +124,7 @@
 #define TOUCH_POS_VALID      (1 << 4) /* Hardware provided a valid X/Y position */
 #define TOUCH_PRESSURE_VALID (1 << 5) /* Hardware provided a valid pressure */
 #define TOUCH_SIZE_VALID     (1 << 6) /* Hardware provided a valid H/W contact size */
-
+#define TOUCH_MAX_POINTS 	15    /* Maximum number of simultaneous touch point supported */
```

Comment:
It's main purpose is for buffer allocation where touch data will be written.
if there is a hardware which has support for less fingers say 10. so, it will detect at max 10 fingers and data will be written for at max 10 indexes. rest 5 index will be empty.
I have added the ifdef config CONFIG_TOUCH_IST415 similar to what we have done in mipi.
so, different touch hardware can define their own TOUCH_MAX_POINTS